backend/src/main/resources/graphql/main.graphqls

```diff
@@ -83,6 +83,16 @@ type User {
   roles: [Role!]! @deprecated(reason: "Users have only one role now")
 }
 
+type UserIdentity {
+  id: ID
+  name: NameInfo
```

Comment:
i would expect us to _not_ expose `name` as its own attribute, the components of it are already laid out right after this line

Reply:
Whereas I was going to ask if we could get rid of the unwrapped name sub-elements yet. 😆 

See also #1219.

Separately,  I think you can probably make this `NameInfo!` since having it be null would be ... surprising.

Reply:
Unfortunately, I think we need to leave `name`, `nameInfo`, and the components for now.  It seems that there is still some usage of each in the frontend.